nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/operations/AbstractTestColdStorageOperation.java

```diff
@@ -66,12 +66,16 @@
     @Inject
     protected CoreFeature coreFeature;
 
+    @Inject
+    protected CoreSession session;
+
     protected void moveContentToColdStorage(CoreSession session, DocumentModel documentModel)
             throws OperationException, IOException {
         try (OperationContext context = new OperationContext(session)) {
             context.setInput(documentModel);
             DocumentModel updatedDocModel = (DocumentModel) automationService.run(context, MoveToColdStorage.ID);
             checkMoveContent(Collections.singletonList(documentModel), Collections.singletonList(updatedDocModel));
+            session.saveDocument(documentModel);
```

Comment:
It there a reason why we are saving the document passed as a parameter. The save is done in the Operation itself and returns the updated document.